### PR TITLE
docs: improve config presets

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Simplify/rewrite sentences
- Fix whitespace
- Fix spelling
- Replace `i.e.` with `e.g.`

## Context:

Helps with issue #7626.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/config-presets.md [hunk 1]
```diff
@@ -42,8 +44,9 @@ The following namespace is used:
 - `gitlab>abc/foo`: gitlab repository `https://gitlab.com/abc/foo` and preset `default`
 - `gitlab>abc/foo:xyz`: gitlab repository `https://gitlab.com/abc/foo` and preset `xyz`
 
-In general, GitHub or GitLab-based preset hosting is easier than npm because you avoid the "publish" step - simply commit preset code to the default branch and it will be picked up by Renovate the next time it runs.
-An additional benefit of using source code hosting is that the same token/authentication can be reused by Renovate in case you want to make your config private.
+It's easier to use GitHub/GitLab based preset hosting, as you avoid the "publish" step that is necessary with a npm package.
```
Comment (HonkingGoose) [inline docs/usage/config-presets.md:47]: Do we want to change this ordered list into a table as well?

Can we maybe copy paste the table from one of my other PRs? (I think the bottom table in the file for PR #7722 might be suitable???)

File: docs/usage/config-presets.md [hunk 2]
```diff
@@ -16,20 +16,22 @@ In short:
...
 ## Implementation Approach
 
-In order to achieve the above goals, preset configs have been implemented to allow a very modular approach - preset configs may be as small as a partial package rule or as extensive as an entire configuration, like an `eslint` config.
+In order to achieve these goals, preset configs allow for a very modular approach - preset configs can be as small as a partial package rule or as large as an entire configuration, like an `eslint` config.
 
 ## Preset Hosting
 
-Presets can be defined using either npm packages, or with GitHub/GitLab repositories. Bitbucket-hosted presets are yet to be implemented.
+Presets can be defined using either npm packages, or with GitHub/GitLab repositories.
+Bitbucket-hosted presets are not supported for now.
```
Comment (JamieMagee) [inline docs/usage/config-presets.md:34]: I would maybe say that something like "presets hosted elsewhere are not supported." Or maybe just remove this sentence entirely?

Comment (HonkingGoose) [reply docs/usage/config-presets.md:34]:
```suggestion
Presets that are hosted elsewhere are not supported.
```

**Review by JamieMagee — DISMISSED** [review]

**Review by rarkins — APPROVED** [review]

**Comment by HonkingGoose** [comment]
Cheers, thanks for merging it in. 😄

**Pull request merged**